### PR TITLE
October changelog

### PR DESCRIPTION
**Pull request by alexfornuto**
## Effect
PR includes the following changes:
- Adds October changelog

## Remaining Work
- [x] Review from @rachelwhitton

### CODE REVIEW

**Comment by rachelwhitton** [comment]
+1

File: source/_changelogs/2017-10-01-October.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+## Platform Improvements
+
+### Custom Upstreams
+We’ve opened access to Custom Upstreams for all organizations on Pantheon. Read more about it on our [blog](https://pantheon.io/blog/announcing-new-pantheon-upstream-workflow){.external}, or [give it a try](/docs/custom-upstream/).
+
+### Terminus 1.6
+If you haven’t updated Terminus recently, check out the <a href="/docs/terminus/updates#changelog" data-proofer-ignore>Terminus Changelog</a> to see what’s new and update to the <a href="/docs/terminus/updates#update-to-the-current-release-" data-proofer-ignore>current release</a>.
+
+### WebP Image Support
+[WebP](https://developers.google.com/speed/webp/){.external}, a new image format created by Google, is now supported by Pantheon’s Global CDN.
```
Comment (ari-gold) [inline source/_changelogs/2017-10-01-October.md]: What does supported mean in this context? How about replacing supported with cached?

File: source/_changelogs/2017-10-01-October.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+### PHP 7.1 & IonCube Decoder
```
Comment (ari-gold) [inline source/_changelogs/2017-10-01-October.md]: Can we move this up to under custom upstreams? I think this is a bigger deal than the two items above it?

**Comment by alexfornuto** [comment]
@ari-gold The copy is already set to go out in the newsletter in its current form. It's too late to change it here.

**Pull request merged**